mypy/messages.py

```diff
@@ -109,6 +109,24 @@
     'Implicit generic "Any". Use \'{}\' and specify generic parameters'  # type: Final
 INCOMPATIBLE_TYPEVAR_VALUE = 'Value of type variable "{}" of {} cannot be {}'  # type: Final
 UNSUPPORTED_ARGUMENT_2_FOR_SUPER = 'Unsupported argument 2 for "super"'  # type: Final
+MULTIPLE_OVERLOADS_REQUIRED = 'Single overload definition, multiple required'  # type: Final
+MISSING_OR_INVALID_SELF_TYPE = \
+    "Self argument missing for a non-static method (or an invalid type for self)"  # type: Final
+ERASED_SELF_TYPE_NOT_SUPERTYPE = \
+    "The erased type of self '{}' is not a supertype of its class '{}'"  # type: Final
```

Comment:
We move towards using double quotes for types, so it would be great to use them here (in the same PR if there are only few tests to update, otherwise in a separate PR).

Reply:
Will do